### PR TITLE
fix(menu): Add specificity to selection group class

### PR DESCRIPTION
**Pull request by williamernest**
fixes: #4169

### CODE REVIEW

File: packages/mdc-menu/mdc-menu.scss [hunk 1]
```diff
@@ -63,13 +63,14 @@
   .mdc-list-item {
     @include mdc-rtl-reflexive-box(padding, left, 56px);
   }
-}
 
-.mdc-menu__selection-group-icon {
-  @include mdc-rtl-reflexive-position(left, 16px);
+  // Extra specificity required because list-item graphic needed to increase specificity.
```
Comment (acdvorak) [inline packages/mdc-menu/mdc-menu.scss]: I don't see `.mdc-list-item__graphic` anywhere on the screenshot test page. How is this related?

**Comment by kfranqueiro** [comment]
I think this is referencing the wrong bug?

Also, should I be able to repro the bug on the new screenshot test page if I un-nest the selector? The page seems to look identical to me in both cases.

**Comment by kfranqueiro** [comment]
Here's the report to update goldens from:

https://storage.googleapis.com/mdc-web-screenshot-tests/travis/2018/12/13/21_10_43_629/report/report.html?utm_source=cli&utm_medium=test_results

I can now repro the bug by reverting the CSS change on the branch.

**Comment by kfranqueiro** [comment]
I noticed the checkmark is misaligned in IE but this seems to be a pre-existing issue to this PR, so I logged #4184.

File: packages/mdc-menu/mdc-menu.scss (hunk 1 above)
Comment (acdvorak) [reply packages/mdc-menu/mdc-menu.scss]: Nit: Can you update this comment to something like:

```js
// Extra specificity required to override `display` property on `mdc-list-item__graphic`.
```
